sdk/eventhubs/azure-messaging-eventhubs/src/main/java/com/azure/messaging/eventhubs/EventHubClientBuilder.java

```diff
@@ -361,6 +363,16 @@ public EventHubClientBuilder prefetchCount(int prefetchCount) {
         return this;
     }
 
+    /**
+     * Set registry serializer
+     * @param objectSerializer serializer
+     * @return updated builder instance
+     */
+    public EventHubClientBuilder objectSerializer(ObjectSerializer objectSerializer) {
```

Comment:
I would have expected to call this `serializer` by default - keen to hear other peoples thoughts though.

Reply:
+1 to Jonathan's thoughts.

Reply:
Not at all attached to this.  I'll update it to serializer.